sympy/utilities/tests/test_lambdify.py

```diff
@@ -39,6 +40,9 @@ def test_single_arg():
     f = lambdify(x, 2*x)
     assert f(1) == 2
 
+def test_print_dummify():
+    assert print_dummify(r'32v2?g#Gmw845h$Wb53wi\phi') == True
+    assert print_dummify(x + y**2) == True
```

Comment:
This test is rather sparse. You only test the `True` case, please also test the other case.